src/forms/inputs/checkbox-group.js

```diff
@@ -145,8 +183,8 @@ function CheckboxGroup(props) {
               ...checkboxInputProps,
```

Comment:
Not a new issue, but can you resolve the accessibility issue for the "with input props specified" story? Looks like we'll need to set the `id` here like we're doing in `RadioGroup` ([line](https://github.com/LaunchPadLab/lp-components/pull/595/files#diff-5c2fdf596caab0ddfe40eef9596137b37bd03591ab1496e366d363d9ad2e9ac2R164)).

Reply:
IMO let's save this for a fast-follow patch release. I'd prefer not to delay this release any longer